### ISSUE TITLE
Idea: Make increasing spawner "Spawn Count" renewable

### ISSUE THREAD

**Issue by James103** [ChosenArchitect/Project-Architect-2#33]
**Is your feature request related to a problem? Please describe.**
Currently, increasing the spawn count of a spawner has been changed to require a Fallen Star from End's Phantasm, a block which, as far as I know, spawns only during world gen at a rate that makes it as rare as Ancient Debris (despite a huge exposure rate). One Fallen Star is consumed for every +1 increase in Spawn Count on a spawner, meaning that one may need to explore the End a lot just to upgrade their spawners to the maximum.

**Describe the solution you'd like**
Either of the following:
* Upgrading the Spawn Count of a spawner requires an item other than Fallen Stars.
* A method to reliably produce or duplicate Fallen Stars is implemented.

**Describe alternatives you've considered**
See above.

**Mod Suggestions**
N/A

**Additional context**
N/A

**Comment by shiftthedev** [comment]
![javaw_1WAl4GnmE6](https://github.com/user-attachments/assets/0a97dfd2-8e3f-4947-a5fe-79d190def863)

its not rare, it literally spawns everywhere in the end.

**Comment by James103** [comment]
> its not rare, it literally spawns everywhere in the end.

However, according to the code in End's Phantsm, there is approximately a 1 in 3 chance that a Fallen Star attempts to generate[^1]. When it does generate, only a single Fallen Star generates[^2].

[^1]: https://github.com/Lyof429/EndsPhantasm/blob/88bdd7a83b8e5b009845c75bc5cd9e719e0d3f5c/src/main/generated/data/phantasm/worldgen/placed_feature/fallen_star.json

[^2]: https://github.com/Lyof429/EndsPhantasm/blob/88bdd7a83b8e5b009845c75bc5cd9e719e0d3f5c/src/main/generated/data/phantasm/worldgen/configured_feature/fallen_star.json

For comparison, according to the Minecraft Wiki, ancient debris generates approximately 1.6 blocks per chunk, not counting any changes made by mods which add/modify content in the Nether[^3].

[^3]: https://minecraft.wiki/w/Ancient_Debris

Thus, if you strip the Nether of all blocks except Ancient Debris, then compare with the Fallen Star generation in the End, you'll notice Fallen Stars being fewer in number than Ancient Debris for the same total area / volume (by a factor of about 4).

In short: Fallen Stars from End's Phantsm _are_ rarer than Ancient Debris in the Nether; you just don't notice that (and instead notice it the other way around) because Fallen Stars are generated in the sky such that an average player can see a couple dozen at once without mining a single block, while Ancient Debris are buried in netherrack and one may need to mine hundreds (even thousands) of blocks to expose a single debris.

**Comment by ChosenArchitect** [comment]
I dont see the issue here lol They are in the sky in the end and can be found and seen easily :D
